api/lanes.py

```diff
@@ -562,3 +590,59 @@ def apply_filters(self, qu, work_model=Work, *args, **kwargs):
         qu = qu.join(work_edition).filter(work_edition.series==self.series)
         qu = qu.order_by(work_edition.series_position, work_edition.title)
         return qu
+
+
+class ContributorLane(QueryGeneratedLane):
+    """A lane of Works written by a particular contributor"""
+
+    ROUTE = 'contributor'
+    MAX_CACHE_AGE = 48*60*60    # 48 hours
+
+    def __init__(self, _db, contributor_name, contributor_id=None):
```

Comment:
I'm a little concerned about the use of database IDs in URLs because contributors can get merged into each other and deleted. A person's database ID at the time the URL is generated might not be their ID at the time the link is followed. 

I think this is okay for now but I would like to move towards using VIAF ID in the URLs, once we have more reliable sources of VIAF IDs, and use the database ID as a fallback for when we don't have VIAF ID.